hack/lib/install.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+#
+# Copyright (C) 2025 ScyllaDB
+#
+
+set -euExo pipefail
+shopt -s inherit_errexit
+
+source "$( dirname "${BASH_SOURCE[0]}" )/bash.sh"
+source "$( dirname "${BASH_SOURCE[0]}" )/kube.sh"
+
+# install-operator installs the Scylla Operator and its dependencies in the cluster through the specified method.
+# $1 - root source path to use. It can either be an absolute file path or a URL.
+function install-operator() {
+  if [[ "$#" -ne 1 ]]; then
+    echo "Missing arguments. Usage: ${FUNCNAME[0]} <source_root>" > /dev/stderr
+    exit 1
+  fi
```

Comment:
arg validation pattern inconsistent with line 16 of `hack/ci-deploy.sh`

https://github.com/scylladb/scylla-operator/pull/3116/commits/7d745c3bd13915520bd1bfad89ab9870e54f7076#diff-b93a0830431b6e0685e8abcc8e25fb21197e6aefa9365a2bb35650c4dce414a6R15

suggestion: unify

Reply:
done